### ISSUE TITLE
Gradle checkGwt task is broken

### ISSUE THREAD

**Issue by GatorQue** [libgdx/libgdx#6525]
Please ensure you have given all the following requested information in your report.

#### Issue details
Running gradle checkGwt task results in build error.

#### Reproduction steps/code
1. Create new project on Windows using libgdx setup tool (I included desktop, android, and html and tools extension)
2. Run gradlew checkGwt from command line
Expected result:
C:\libgdx\cavegame>gradlew checkGwt

> Task :html:checkGwt
C:\libgdx\cavegame\html\.
../android/assets
Copying resources from ../android/assets to war/
C:\libgdx\cavegame\html\..\android\assets
C:\libgdx\cavegame\html\assets
C:\libgdx\cavegame\html\.
../android/assets
Copying resources from ../android/assets to war/
C:\libgdx\cavegame\html\..\android\assets
C:\libgdx\cavegame\html\assets

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.7.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 7s
4 actionable tasks: 2 executed, 2 up-to-date

Actual result:
C:\libgdx\cavegame>gradlew checkGwt

> Task :html:checkGwt FAILED
Loading inherited module 'com.cosmicrover.cavegame.GdxDefinition'
   Loading inherited module 'CaveGame'
      [ERROR] Unable to find 'CaveGame.gwt.xml' on your classpath; could be a typo, or maybe you forgot to include a classpath entry for source?

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':html:checkGwt'.
> Process 'command 'C:\libgdx\jdk-15\bin\java.exe'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.7.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 1s
3 actionable tasks: 1 executed, 2 up-to-date

#### Version of LibGDX and/or relevant dependencies
LibGDX 1.10.0

#### Please select the affected platforms
- [] Android
- [ ] iOS
- [X] HTML/GWT
- [ ] Windows
- [ ] Linux
- [ ] MacOS

#### Solution:
html\build.gradle should have the following line added near similar lines at the bottom:
tasks.checkGwt.dependsOn(addSource)

**Comment by tommyettinger** [comment]
The checkGwt task is, uh, rarely used, as you have discovered. I've fixed this in gdx-liftoff, though I haven't released a new version yet (the linked commit just above this comment shows the changes I made). I think a new libGDX version was planned for soon-ish, and that will be a good time to release a new version of that alternative setup tool. I don't really know much about how the official gdx-setup is currently versioned and released, but this should be pretty easy to add to it.
